src/js/control-bar/seek-to-live.js

```diff
@@ -61,9 +61,13 @@ class SeekToLive extends Button {
    */
   updateLiveEdgeStatus(e) {
     if (this.player_.liveTracker.behindLiveEdge()) {
+      this.setAttribute('aria-disabled', false);
       this.removeClass('vjs-at-live-edge');
+      this.controlText('Seek To LIVE');
     } else {
       this.addClass('vjs-at-live-edge');
+      this.controlText('At LIVE edge');
```

Comment:
"Playing Live", perhaps? As I mentioned, I don't think "edge" is meaningful outside the technical world.

Reply:
Or "Seek to Live, currently playing Live", which would then be announced as "button, disabled".